### DISCUSSION TITLE
How are you going to handle migrations?

### DISCUSSION THREAD

**Discussion by Dorfeuheinz** [vinniefranco/exandra#16] [Q&A] [answered]
I was wondering how are you planning on handling migrations. Since I was looking for ways to do that too, the initial thought that came to mind was to just use ecto migrations & write macros to read the migration files & generate the corresponding cassandra queries. How are you planning on doing it? If you have a better solution kindly inform me as well since right now im just doing everything manually without any migrations. I might even contribute if the above idea turns out to be a good solution. 
Thanks

**Comment by whatyouhide** [comment, marked as answer]
@Dorfeuheinz the `main` branch now supports migrations without having to do anything special, you can use regular Ecto migrations the way you would with other adapters. 👍
